app/helpers/admin/admin_helper.rb

```diff
@@ -59,4 +59,13 @@ def render_custom_links
   def cms_title
     @page.try(:name) || "Slices CMS"
   end
+
+  # Create a link to view the current page on the live site
+  #
+  # @!visibility private
+  def link_to_view_page(page, options = {})
+    options.reverse_merge!(target: 'site', class: 'view-page')
```

Comment:
The old version had `target: '_blank'`, so is this change intentional? The commit is about removing the double slash so can't tell...

Same question about the `return unless page.active?` bit too. If we're changing other behaviour of the method, it would be nice to split into separate commits to keep it clear.